script.js

```diff
@@ -1,12 +1,137 @@
 //You can edit ALL of the code here
+const rootElem = document.getElementById("root");
+const dropDown = document.getElementById("list");
+let allEpisodes = getAllEpisodes();
+const searchBar = document.getElementById("search");
+const display = document.getElementById("display");
+const btn = document.getElementById("btn")
+let allShows = getAllShows();
+const showDropDown = document.getElementById("showList")
+let initialId = "82";
+let shows = [];
+
+let showId = allShows.map(el=> el.id);
+
+ function showDisplayer(showId){
+  fetch(`https://api.tvmaze.com/shows/${showId}/episodes`)
+  .then(response => response.json())
+  .then((data) =>{ shows = [...data] 
+  makePageForEpisodes(data)
+  allEpisodes = data
+  displayer()
+  dropDownSelection(data)})
+  .catch((error) => console.log(error));
+} 
+
+function makePageForEpisodes(episodeList) {
+  rootElem.innerHTML = "";
+  display.innerText = `Display: ${episodeList.length}/${allShows.length}`;
+  episodeList.forEach(e => {
+  const boxBody = document.createElement('div');
+  boxBody.id = "boxBody";
+  const title = document.createElement('h3');
+  title.id = "title";
+  const imgBox = document.createElement('img');
+  imgBox.id = "imgBox";
+  const para = document.createElement('p');
+  para.id = "para"
+  rootElem.appendChild(boxBody);
+  boxBody.append(title,imgBox,para);
+  title.innerText = `${e.name}`
+  +`-S${e.season.toString().padStart(2,'0')}E${e.number.toString().padStart(2,'0')}`;
+  imgBox.src = e.image.medium;
```

Comment:
I think some episodes / seasons don't have images, which gives you an error in the console. Add an `if` statement to check if there is no image and show a default image instad.